src/language/lang_it.php

```diff
@@ -0,0 +1,659 @@
+    'Spiacente il modulo non è stato salvato. Si prega di correggere gli errori e salvare nuovamente.',
+  'choose_logo' =>
+    'scegli_logo',
+  'captured_' =>
+    'catturato_',
+  'flag_owner_' =>
+    'bandiera_proprietario_',
+  'INACTIVE' =>
+    'INATTIVO',
+  'PTS' =>
+    'PTS',
+  'category' =>
+    'categoria',
+  'capture_' =>
+    'cattura_',
+  'Insert your answer' =>
+    'Inserisci la tua risposta',
+  'Request Hint' =>
+    'Richiedi Aiuto',
+  'Submit' =>
+    'Invia',
+  'hint_' =>
+    'Aiuto_',
+  'first_capture' =>
+    'prima_cattura',
+  'completed_by' =>
+    'completata_da',
+  'scoreboard_' =>
+    'scoreboard_',
+  'filter_' =>
+    'filtro_',
+  'rank_' =>
+    'grado_',
+  'team_name_' =>
+    'squadra_nome_',
+  'quiz_pts_' =>
+    'quiz_pts_',
+  'flag_pts_' =>
+    'bandiera_pts_',
+  'base_pts_' =>
+    'base_pts_',
+  'total_pts_' =>
+    'total_pts_',
+  'team_' =>
+    'squadra_',
+  'team_members' =>
+    'squadra_membri',
+  'base_pts' =>
+    'base_pts',
+  'quiz_pts' =>
+    'quiz_pts',
+  'flag_pts' =>
+    'bandiera_pts',
+  'total_pts' =>
+    'total_pts',
+  'Tool bars are located on all edges of the gameboard. Tap a category to expand and close each tool bar.' =>
+    'barre degli strumenti si trovano su tutti i bordi del pannello di gioco. Toccare una categoria per espandere e chiudere ogni barra degli strumenti.',
+  'Tool_Bars' =>
+    'Tool_Bars',
+  'Tap the "Game Clock" to keep track of time during gameplay. Don’t let time get the best of you.' =>
+    'premi su "Orologio di giooc" per tenere traccia del tempo durante il gioco. Non lasciare che il tempo abbia la meglio su di voi.',
+  'Game_Clock' =>
+    'Game_Clock',
+  'Countries marked with an ' =>
+    'Paesi contrassegnati con un  ',
+  'are captured by you.' =>
+    'sono stati catturati da te.',
+  ' are owned by others.' =>
+    ' sono di altri.',
+  'Captures' =>
+    'Cattura',
+  'Tap Plus[+] to Zoom In. Tap Minus[-] to Zoom Out.' =>
+    'premi sul più[+] per avvicinarti. premi su meno[-] per allontanarti.',
+  'Click and Drag to move left, right, up and down.' =>
+    'clicca e trascina per muoverti a sinistra, destra, su e giù.',
+  'Zoom' =>
+    'Zoom',
+  'Tap Forward Slash [/] to activate computer commands. A list of commands can be found under "Rules".' =>
+    'primi barra[/] per attivare i comandi del computer. una lista di comandi la puoi trovare sotto "Regole".',
+  'Command_Line' =>
+    'Command_Line',
+  'Click "Nav" to access main navigation links like Rules of Play, Registration, Blog, Jobs & more.' =>
+    'clicca su "Nav" per accedere alla navigazione principale come regole del gioco, registrazione, blog, lavori e altro.',
+  'Track your competition by clicking "scorboard" to access real-time game statistics and graphs.' =>
+    'Tieni traccia della competizione cliccando su "Tabella punti" per accedere alla statistice e grafici in tempo reale del gioco.',
```

Comment:
Tieni traccia della competizione cliccando su "Tabella punti" per accedere alla statistiche e grafici in tempo reale del gioco.